tests/azure/storageaccount_test.go

```diff
@@ -0,0 +1,57 @@
+package azure_test
+
+import (
+	"fmt"
+	"math/rand"
+	"os"
+	"testing"
+
+	"github.com/NeowayLabs/klb/tests/lib/azure"
+	"github.com/NeowayLabs/klb/tests/lib/azure/fixture"
+	"github.com/NeowayLabs/klb/tests/lib/nash"
+)
+
+func genStorageAccountName() string {
+	return fmt.Sprintf("klbstortests%d", rand.Intn(1000))
+}
+
+func testStorageAccountCreate(t *testing.T, f fixture.F) {
+	genstorage := genStorageAccountName()
+	nash.Run(
+		f.Ctx,
+		t,
+		"./testdata/create_storage_account.sh",
+		f.ResGroupName,
+		genstorage,
+		f.Location,
+	)
+	storage := os.Getenv("STORAGE_ACCOUNT_NAME")
```

Comment:
@tiago4orion We did not tought on a better way to get a value produced from the shell. AFAIK it would be:

- env var
- stdout / stderr
- a file

Now that I was thinking, since we are running the shell embedded on Go, it seems that it could be a simple variable if there is an API on nash to read variable values from the Go code.

Another idea is something supported on Lua when it is embedded, on Lua if a script returns something, the returned value is returned to the caller of the script. This is also how importing modules work on Lua, the module system is just a function that calls the script and stores the returned value on a table.

But AFAIK, nash does not have that. Is there any other option ? From the ones we considered the env var seemed the better one.